src/types/categories/interfaces/categories.interfaces.ts

```diff
@@ -1,5 +1,15 @@
+import { CommonEntityFields } from '~/types/common/common.index'
 import { RequestParams } from '~/types/services/types/services.types'
 
 export interface CategoriesParams extends RequestParams {
   name: string
 }
+
+export interface CreateCategoriesParams {
+  name: string
+}
+
+export interface CreatedCategory extends CommonEntityFields {
```

Comment:
you can use "Categories" interface, it looks the same

Reply:
> you can use "Categories" interface, it looks the same

You are right! Thanks for the advice